cmd/gokeyless/gokeyless.go

```diff
@@ -54,31 +55,32 @@ func init() {
 func main() {
 	flag.Parse()
 
-	s, err := server.NewServerFromFile(certFile, keyFile, caFile, net.JoinHostPort("", port), "")
-	if err != nil {
-		log.Warningf("Could not create server. Running initializeServer to get %s and %s", keyFile, certFile)
-
-		// If an existing CSR (and associated key) exists, don't
-		// proceed/overwrite the key.
-		if _, err := os.Stat(csrFile); !os.IsNotExist(err) {
-			log.Fatalf("an existing CSR was found at %q: remove once a signed certificate has been installed.", csrFile)
```

Comment:
The flag of `-manual-activation` now would check csr and key file, and matching them through public key. Also, we make it clear that `-manual-activation` won't start the server.